config/runtime/tests/fluster-chromeos.jinja2

```diff
@@ -0,0 +1,70 @@
+{%- include 'chromeos/base.jinja2' %}
+
+{%- if testsuite -%}
```

Comment:
Error message [in the relevant Node](https://staging.kernelci.org:9000/viewer?node_id=66d8184915e96f36b9e14b22) (`block sequence entries are not allowed in this context\n  in \"\", line 157, column 27`) was caused by right-stripping whitespace from the template - rendered TestJob definition looked as follows:

```
      password: "test0000"- test:
```

instead of:

```
      password: "test0000"
- test:
```

```suggestion
{%- if testsuite %}
```

(or just run `sed -i 's/-%}/%}/' config/runtime/tests/fluster-chromeos.jinja2` to get rid of all of them)

Reply:
@denisyuji I probably should have made this more explicit: *all* `rstrips()` (dashes before closing tag: `-%}`) need to go away.

I dropped one in my code suggestion and advised to run `sed` on this file so that all of them would be removed.

Reply:
This has been addressed by https://github.com/kernelci/kernelci-core/pull/2658